Server/src/exceptions/httpException.ts

```diff
@@ -0,0 +1,12 @@
+// https://wanago.io/2018/12/17/typescript-express-error-handling-validation/
```

Comment:
Appreciate putting in the link! vere nice.
You might wanna take a look into this:
https://tsed.io/docs/exceptions.html
https://github.com/TypedProject/tsed/tree/production/packages/exceptions
instead of implementing our own we could use this lib